common/proc_allowing-external-access-knative-services-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+[id='allowing-external-access-knative-services-OCP-4x_{context}]
+= Allowing external access to Knative services on OCP 4.0
+
+Use a configured hostname to access Knative services on the OpenShift 4.0 cluster
```

Comment:
[Suggestion] @joaedwar what do docs guidelines say about intro paragraphs, if anything? Would it make more sense to have something like "This section shows how to configure..." or is this different to our style guide for other projects, e.g. AMQ?

Reply:
I think the intro paragraph might need more info; maybe we should explain why a user might need a configured hostname. Looking at other documentation examples (e.g. AMQ streams), including something like "This section shows how..." is not the convention. I have seen such text in the abstract for sections, but the introductory paragraphs for procedure modules usually have a more active and user-centric tone. Also, the OCP 3.11 post-installation tasks procedure module has a similar introductory paragraph.

Reply:
That sounds good - do you want to raise a task in Jira to track work on the configured hostname info internally so we remember to do this?
Thanks so much for the clarity on conventions, I haven't had as much time as I should have to look at these.